collect_app/src/main/res/layout/main_menu_button.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginVertical="@dimen/margin_small"
+    android:layout_marginHorizontal="@dimen/margin_standard"
+    android:elevation="4dp"
```

Comment:
Again going by the [Elevated Button spec](https://m3.material.io/components/buttons/specs#7026d152-e4c1-4de9-9d06-5631c09c3f4b), this should actually be `1dp` rather than `4dp` (looking at the different levels in the [Elevation docs](https://m3.material.io/styles/elevation/tokens)). Annoyingly, they've also left this out of Material Components although this could be because the system is pretty simple: Level 0 is 0dp, Level 1 is 1dp and so on. 

We could create these levels in `dimens.xml` or just use `1dp`. Either works for me.

Reply:
Done.